src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.h

```diff
@@ -1,17 +1,29 @@
 #ifndef Z_OBJ_TOKEI_TOBIRA_H
 #define Z_OBJ_TOKEI_TOBIRA_H
 
+#define OBJTOKEITOBIRA_GET_TYPE(thisx) ((thisx)->params & 1)
+#define OBJTOKEITOBIRA_PARAM(type) (type)
```

Comment:
```suggestion
#define OBJTOKEITOBIRA_PARAMS(type) (type)
```
Kind of nitpicky, but plural seems better for these bit packing params macros.